### ISSUE TITLE
Arquivo remessa, baixa boleto

### ISSUE THREAD

**Issue by gamadoleo** [thiagohenrique0810/GARB-Gerador-de-Arquivos-de-Remessa-Bradesco#5]
Bom dia, Olá alguem já teve a necessidade da geração de arquivo remessa para baixa, boleto, tipo assim, digamos que você recebeu o pagamento em mãos, e você necessita dar baixa no boleto para que no sistema bancario conste como pago??

**Comment by thiagohenrique0810** [comment]
sim, você nas configurações do DETALHE pode informar que a remessa com o boleto que você configurou é um comando de baixa.
